encord/objects/ontology_labels_impl.py

```diff
@@ -778,7 +786,7 @@ def add_object_instance(self, object_instance: ObjectInstance, force: bool = Tru
         self._add_to_frame_to_hashes_map(object_instance, frames)
 
     def add_classification_instance(
-        self, classification_instance: ClassificationInstance, force: bool = False, use_range: bool = False
+        self, classification_instance: ClassificationInstance, force: bool = False
```

Comment:
Sneaky fix here from previous PR.

Left this unused `use_range` property here. It is not needed anymore.